### PR TITLE
Reorganize docs

### PR DESCRIPTION
**Pull request by mtreinish**
### Summary

PR #1224 attempted to leave bread crumbs in the documentation to get
people used to the new package organization and split between Qiskit and
downstream packages (applications, experiments, etc). However, in
practice this has been causing more confusion for users than helping.
This primarily stemming from the use of the word "Experiments" to
describe ignis instead of pointing to qiskit-experiments. This reverts
that PR and starts to starts to reoganize things to be a bit clearer.

### Details and comments

TODO:

- [x] update top navbar to list experiments too

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -37,6 +37,50 @@ around the world by beginners, hobbyists, educators, researchers, and commercial
...
+  :description: Application of Quantum Computing for Natural Sciences
+  :header: Qiskit Nature
+  :button_link: https://qiskit.org/documentation/nature/
+  :button_text: Qiskit Nature
+
+.. raw:: html
+
+  </div>
+
+Interested in running characterization, calibration, and verification experiments?
```
Comment (coruscating) [inline docs/index.rst]: This seems a bit wordy since the "characterization, calibration, and verification" is already in the description below. What about just "Interested in running experiments on real qubits?"

File: docs/index.rst [hunk 2]
```diff
@@ -37,6 +37,50 @@ around the world by beginners, hobbyists, educators, researchers, and commercial
 
   </div>
 
+Interested in Applications of Quantum Computing?
```
Comment (coruscating) [inline docs/index.rst]: This section fully capitalizes titles and descriptions, while the next sections only capitalize the beginnings

Comment (mtreinish) [reply docs/index.rst]: Good catch, done in: https://github.com/Qiskit/qiskit/pull/1323/commits/5a7bd9f6287e6ce7482a9c3cdfcc7dcef09c4a59

File: docs/index.rst (hunk 1 above)
Comment (mtreinish) [reply docs/index.rst]: Yeah, that would look better, done in: https://github.com/Qiskit/qiskit/pull/1323/commits/5a7bd9f6287e6ce7482a9c3cdfcc7dcef09c4a59

**Review by coruscating — DISMISSED** [review]
LGTM

File: README.md [hunk 3]
```diff
@@ -28,18 +28,17 @@ The four elements of Qiskit are the essential parts that give Qiskit its power
...
+| [**Qiskit Ignis**](https://github.com/Qiskit/qiskit-ignis)  |  [![Build Status](https://img.shields.io/travis/com/Qiskit/qiskit-ignis/master.svg?)](https://travis-ci.com/Qiskit/qiskit-ignis) |  [![](https://img.shields.io/github/release/Qiskit/qiskit-ignis.svg?)](https://github.com/Qiskit/qiskit-ignis/releases) | [![](https://img.shields.io/github/forks/Qiskit/qiskit-ignis.svg?)](https://github.com/Qiskit/qiskit-ignis) |
+| [**Qiskit Aqua**](https://github.com/Qiskit/qiskit-aqua) |  [![Build Status](https://img.shields.io/travis/com/Qiskit/qiskit-aqua/master.svg?)](https://travis-ci.com/Qiskit/qiskit-aqua) |  [![](https://img.shields.io/github/release/Qiskit/qiskit-aqua.svg?)](https://github.com/Qiskit/qiskit-aqua/releases) | [![](https://img.shields.io/github/forks/Qiskit/qiskit-aqua.svg?)](https://github.com/Qiskit/qiskit-aqua) |
```
Comment (ajavadia) [inline README.md]: I don't think we should have these elements now in the README. We should have terra, aer, experiments, dynamics, nature, optimization, machine-learning, finance.

image

File: docs/index.rst [hunk 4]
```diff
@@ -87,18 +130,18 @@ Interested in quantum hardware design?
...
-  Qiskit (Terra) <apidoc/terra>
-  Qiskit Simulator (Aer) <apidoc/aer>
-  Qiskit Experiments (Ignis) <apidoc/ignis>
-  Qiskit Application Modules (Aqua) <apidoc/aqua>
-  Qiskit IBM Quantum (Provider) <apidoc/ibmq-provider>
+  Qiskit Terra <apidoc/terra>
+  Qiskit Aer <apidoc/aer>
+  Qiskit Ignis (deprecated) <apidoc/ignis>
+  Qiskit Aqua (deprecated) <apidoc/aqua>
+  Qiskit IBM Quantum Provider <apidoc/ibmq-provider>
```
Comment (ajavadia) [inline docs/index.rst:137]: Can we list qiskit-experiments, qiskit-dynamics here? 

Also since qiskit-ibm will replace the qiskit-ibmq-provider, is there a link to those docs? (maybe from within the partners)

Comment (mtreinish) [reply docs/index.rst:137]: This is for qiskit documentation/api reference, the other packages are downstream consumers of qiskit and have their own docs. The way things are split now the top navbar has the links to related projects (partners, applications, etc) and ideally is shared between all the docs for any project in Qiskit. The side bar is reserved for qiskit documentation (and is just generated from the main toctree). We can change that if you want, but then the division between project boundaries might get a bit fuzzy.

File: README.md (hunk 3 above)
Comment (mtreinish) [reply README.md]: Sure, I'll reorganize the table to uinclude more and document what's in the metapackage.

File: docs/index.rst (hunk 4 above)
Comment (mtreinish) [reply docs/index.rst:137]: The qiskit-ibm docs aren't published yet because they haven't released yet. Once there is a release the docs will be published to https://qiskit.org/documentation/partners/ibm/

File: README.md (hunk 3 above)
Comment (mtreinish) [reply README.md]: ok I updated this in: https://github.com/Qiskit/qiskit/pull/1323/commits/3f82456c156d8de2ffe14fec162a0777f35ca2e9

**Comment by woodsp-ibm** [comment]
Should we have the order of the listing of the application modules in the readme and in the documentation dropdown Applications menu consistent with the order in the index.rst. Otherwise, currently I think the orderings are all different

File: README.md [hunk 5]
```diff
@@ -22,23 +22,52 @@ $ pip install qiskit
...
+| [**Qiskit Machine Learning**](https://github.com/Qiskit/qiskit-machine-learning) | [![](https://img.shields.io/github/release/Qiskit/qiskit-machine-learning.svg?)](https://github.com/Qiskit/qiskit-machine-learning/releases) | [![](https://img.shields.io/github/forks/Qiskit/qiskit-machine-learning.svg?)](https://github.com/Qiskit/qiskit-machine-learning) |
+| [**Qiskit Nature**](https://github.com/Qiskit/qiskit-nature) |  [![](https://img.shields.io/github/release/Qiskit/qiskit-nature.svg?)](https://github.com/Qiskit/qiskit-nature/releases) | [![](https://img.shields.io/github/forks/Qiskit/qiskit-nature.svg?)](https://github.com/Qiskit/qiskit-nature) |
+| [**Qiskit Experiments**](https://github.com/Qiskit/qiskit-experiments) |  [![](https://img.shields.io/github/release/Qiskit/qiskit-experiments.svg?)](https://github.com/Qiskit/qiskit-experiments/releases) | [![](https://img.shields.io/github/forks/Qiskit/qiskit-experiments.svg?)](https://github.com/Qiskit/qiskit-experiments) |
+
+### Using quantum services
+
+If you're interested in using quantum services (ie using a real quantum
```
Comment (ajavadia) [inline README.md]:
```suggestion
If you are interested in using quantum services (i.e. using a real quantum
```

File: README.md [hunk 6]
```diff
@@ -22,23 +22,52 @@ $ pip install qiskit
 See [install](https://qiskit.org/documentation/getting_started.html) Qiskit for detailed instructions, how to use virtual environments, and
 build from source standalone versions of the individual Qiskit elements and components.
 
-## Qiskit Elements
+## Qiskit Packaging
 
-The four elements of Qiskit are the essential parts that give Qiskit its power
+The Qiskit project is made up of many components that all serve a particular purpose,
+this repository contains a meta-package that will install the following components
```
Comment (ajavadia) [inline README.md]:
```suggestion
The Qiskit project is made up of many components that all serve a particular purpose. 
This repository contains a meta-package that will install the following components:
```

**Review by ajavadia — DISMISSED** [review]
